src/fiatExchanges/SelectProvider.tsx

```diff
@@ -158,39 +161,80 @@ export default function SelectProviderScreen({ route, navigation }: Props) {
     asyncProviders.result?.externalProviders
   )
 
+  const supportOnPress = () => navigate(Screens.SupportContact)
+
+  const switchCurrencyOnPress = () =>
+    navigate(Screens.FiatExchangeCurrency, {
+      flow:
+        route.params.flow === CICOFlow.CashIn ? FiatExchangeFlow.CashIn : FiatExchangeFlow.CashOut,
+    })
+
   const exchanges = asyncExchanges.result ?? []
+  const legacyMobileMoneyProviders = asyncProviders.result?.legacyMobileMoneyProviders
+
+  const anyProviders =
+    normalizedQuotes.length ||
+    !(coinbaseProvider?.restricted ?? true) ||
```

Comment:
I think you need to add more checks to determine if the Coinbase pay section will show up. [These](https://github.com/valora-inc/wallet/blob/awtsui/no-payment-option/src/fiatExchanges/CoinbasePaymentSection.tsx#L39) are that checks that are currently made in the Coinbase payment section

Reply:
The code looks good to me otherwise!

Reply:
Yup, I agree. Coinbase pay section checks have been moved to SelectProvider.